doc/governance/index.rst

```diff
@@ -0,0 +1,116 @@
+Ceph Governance
+---------------
+
+Key principles
+==============
+
+  * Decision-making is consensus-driven by those who show up.
+  * Leadership roles are defined primary by responsibility, not prestige or seniority.
+    * It is normal and healthy for these roles to be passed on to others
+  * The role of developers is ultimately to serve the users.
+
+Bodies
+------
+
+Ceph Executive Council
+======================
+
+  * [3 or 5] people
+  * selected by steering committee during leadership summit
+    * candidates self-nominate or are nominated by other members
+    * discussion of how roles/responsibilities may be delegated
+    * propose 1 or more council sets
+    * ranked-choice vote by steering committee
+  * 1 year term (i.e., a release cycle) (or staggered longer terms?)
```

Comment:
```suggestion
  * 2 year staggered term. 
```
I like a staggered term as it keeps institutional knowledge in place and avoids major down times during elections. Downside of  a two year staggered election is that we'll going to have elections every 8 months, which is a lot as well as making it hard to celebrate elections

Reply:
In the CLT meeting we were still undecided on some of this model. I was definitely on the frequent, lower-stakes elections side (I suggested 8-month terms with a single seat up every 4 months) rather than trying to have long-term elections lasting across a release or two.